admin/sql/create_tables.sql

```diff
@@ -8,4 +8,15 @@ CREATE TABLE "user" (
 );
 ALTER TABLE "user" ADD CONSTRAINT user_musicbrainz_id_key UNIQUE (musicbrainz_id);
 
+CREATE TABLE listens (
+  id              SERIAL,
+  uid             VARCHAR NOT NULL,
+  date            TIMESTAMP,
+  timestamp       INT NOT NULL,
+  artist_msid     UUID,
+  album_msid      UUID,
+  recording_msid  UUID,
+  json            VARCHAR
```

Comment:
Ah yes, correct. @pinkeshbadjatiya: Please make the artist and recording_msid NOT NULL.

If it isn't indexed, it will still create an artist msid and rcording_msid, so those should be required.